public/static/docs/command-reference/update.md

```diff
@@ -75,3 +73,14 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+To update from and also lock to a specific revision of a <abbr>data
+artifact</abbr>, we may use the `--rev` option this time:
```

Comment:
Nice addition to the examples! How about:

`To force an update from a specific version of  a data artifact, we may specify a Git revision with the `--rev` option:`

for this sentence? And please wrap `<abbr>data artifact</abbr>` in the first paragraph of this whole **Examples** section instead (thanks for catching this.)

Reply:
(p.s. I think let's avoid the term "lock" here which results confusing given how it's previously used, and given our `dvc lock` command, among other types of locking in DVC.)